lib/y2_installbase.pm

```diff
@@ -98,10 +98,10 @@ sub deal_with_dependency_issues {
         send_key_until_needlematch 'packages-section-selected', 'tab';
         send_key 'ret';
     }
-    if (get_var("WORKAROUND_DEPS")) {
+    if (check_var("WORKAROUND_DEPS", '1')) {
         y2_logs_helper::workaround_dependency_issues;
     }
-    elsif (get_var("BREAK_DEPS")) {
+    elsif (check_var("BREAK_DEPS", '1')) {
         y2_logs_helper::break_dependency;
     }
```

Comment:
If you change , then need cover all scenario, such as: zdup
lib/y2_installbase.pm:101:    if (get_var("WORKAROUND_DEPS")) {
tests/installation/change_desktop.pm:35:    if (check_screen('dependency-issue', 5) && get_var("WORKAROUND_DEPS")) {
tests/installation/zdup.pm:110:            if (get_var("WORKAROUND_DEPS")) {